### ISSUE TITLE
Change wording n/a on attribute's default value 

### ISSUE THREAD

**Issue by madscollin** [mjmlio/mjml#2279]
**Is your feature request related to a problem? Please describe.**
I do not understand why the alt attribute is _n/a_ on `<mj-image> `- For best practices accessibility, alt tags should be shown on images. But it is not possible at the moment.

**Describe the solution you'd like**
Allow alt tags on the mj-image tag. 
`<mj-image src="..." alt="my alt description"></mj-image> `

**Compiles to -->**

`<img src="..." alt="my alt description" />`

**Comment by iRyusa** [comment]
Well n/a in this case just mean there's no default value available for this attribute. If it's listed then it's supported.

Might need to reword those `n/a` to something else (none could be interpreted as css value `none` available for few tags).

I'm still working on a script to generate those tables directly from the component, I'll take a look to change `n/a`.

**Comment by mohamedsalem401** [comment]
I can make a simple python program to change 'n/a' in the documentation to '-' so it will be less ambiguous if it will help.

**Comment by mohamedsalem401** [comment]
Also, I can uniform descriptions of attributes across different components.

### Example before :
![image](https://user-images.githubusercontent.com/67981832/123187427-705b0f00-d49a-11eb-990c-9949897d7f02.png)
![image](https://user-images.githubusercontent.com/67981832/123187453-7bae3a80-d49a-11eb-939b-79d2615edf76.png)

### Become:
![image](https://user-images.githubusercontent.com/67981832/123187427-705b0f00-d49a-11eb-990c-9949897d7f02.png)
![image](https://user-images.githubusercontent.com/67981832/123187427-705b0f00-d49a-11eb-990c-9949897d7f02.png)

**Comment by iRyusa** [comment]
I think it would be better to work on a script that generate attribute table for a given component. I don't know where we could store the description tho

**Comment by dazgreer** [comment]
Closing as documentation received an update at the end of last year.
